hazelcast/src/test/java/com/hazelcast/cache/CacheConfigTest.java

```diff
@@ -203,6 +199,19 @@ public void defaultCacheTest() {
     }
 
     @Test
```

Comment:
No, we discussed it with Peter already and defined that try/catch will be better.